meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -25,3 +28,8 @@ do_install_append() {
 
     ln -s /var/run/resolvconf/interface/NetworkManager ${D}/etc/resolv.dnsmasq
 }
+
+do_deploy_append() {
+    mkdir -p "${DEPLOY_DIR_IMAGE}/system-connections/"
+    install -m 0600 "${WORKDIR}/resin-sample" "${DEPLOY_DIR_IMAGE}/system-connections/"
+}
```

Comment:
You also need to add this to the bbappend:

+addtask deploy before do_package after do_install
+do_deploy[dirs] += "${DEPLOYDIR}/${PN}"